css-ui-4/Overview.bs

```diff
@@ -1785,13 +1785,16 @@ so this specification allows flexibility.
 </div>
 
 While the way most elements in a document look can be fully controlled by CSS,
-controls are typically rendered by UAs using native UI controls of the host operating system,
+<a>controls</a> are typically rendered by UAs using native UI controls of the host operating system,
 which can neither be replicated nor styled using CSS.
 
+The term <dfn>control</dfn> denotes the following [[HTML]] elements:
```

Comment:
Is it good to be quite this specific? What if things change? Are there disagreements even on some in terms of support? Could you say instead "Examples of controls in HTML are: ....."

Reply:
This definition, like the spec itself, is meant to be canonical. Ideally, if things change, they'd start by changing here. Even if we find ourselves updating the definition reactively, it seems to me that completeness is a goal of the standard. From that perspective, suggesting a partial list is less helpful than defining a complete set.

Reply:
is <summary>/<details>  not a control then?

Reply:
Good question! I'll defer to the editor on that.

Reply:
I'm thinking that maintaining this in a css spec is the wrong place. Tracking how the HTML spec changes will never be perfectly synchronized. Ideally, we'd just refer to a term defined in the HTML spec, and they'd update it when they add new elements. However, there doesn't seem to be one already.

Reply:
> Ideally, we'd just refer to a term defined in the HTML spec

Doesn't the existing term [_widget_](https://html.spec.whatwg.org/multipage/rendering.html#widgets) satisfy this criteria? BTW, it already includes `<details>`.